demo/demo_secondary.py

```diff
@@ -288,7 +288,7 @@ def update_cycle():
     'http://' + str(_primary_host) + ':' + str(_primary_port))
 
   # Download the time attestation from the Primary.
-  time_attestation = pserver.get_last_timeserver_attestation()
+  time_attestation = pserver.get_time_attestation_for_ecu(_ecu_serial)
```

Comment:
The pull request summary explains:

> When in DER mode (default), the function exposed in the demo
> Primary's XMLRPC interface to return the latest timeserver
> attestation for use by the demo Secondary **was not correctly
> wrapping the DER encoded attestation in an xmlrpc Binary()
> object for transmission.**

The pull request summary might be easier to understand if you simply say that `get_last_timeserver_attestation()` doesn't work in Python 2.7.14.

When in DER mode, `time_attestation` is overwritten on line 298, so the value returned by `get_last_timeserver_attestation()` doesn't matter.  I was trying to use the PR summary to trace the cause of the error but realized too late that the the actual issue is in `get_last_timeserver_attestation()`.

Reply:
Line 298 overwrites X with X.Y, so X matters. Code used to break on line 291 in Python 2.7.14, so we didn't get there.

I could probably have been clearer in the PR summary by explaining more - agreed. It is not the case, though, that get_last_timeserver_attestation() doesn't work in Python 2.7.14.